### PR TITLE
Add coinbase connection 

### PR DESCRIPTION
**Pull request by hnguyen1179**
Changes: 

* Added coinbase connector
* Changes to wallet dialog:  
    * Noticed that the `Close wallet connection` button wasn't actually deactivating the current wallet connection, but just closing the modal which was different from the expected behavior of the main app and so I made the change to also deactivate the current wallet connection on click of that button
* Converted [proposals.tsx](https://github.com/element-fi/elf-council-frontend/pull/648/commits/8ae0ea641a51bc47dba08280f423cacb26aaf441#diff-c6af1346b5251868e9de9317990afa205e7aa82ba3b6f3a31d73929ea4dc36c5) from using SSR to SSG/ISR with a refresh interval of 60s
*  Code clean up to overview page
   * Removed usage of `getRecentDelegators` now that the banner is removed 
* General code clean up; fixed up all of the build warnings
   * Added linter exception comments for the sparse usage of `any`
   * Commented out some unused, assigned variables in [initializeGovernance.ts](https://github.com/element-fi/elf-council-frontend/pull/648/commits/929b0eb9d0e216e2ec9494343ef975de2f6d3818#diff-1407980081045d58c445582f3aa1d1c8ff54b6a9102475ca6ba88476c9a67c5b) (this was probably a WIP for testing)
   * Removed unused usage of `whiteText` in favor of a more general `textClassName` prop in [LabeledStat.tsx](https://github.com/element-fi/elf-council-frontend/pull/648/commits/5a8a81e1d8afd734f4135c1567a69ad88cfaf62a#diff-74332d6195374f894e18aa5448f4fcae3253d38a814384c18270134b5aede4b2)
   * Removed some unused imports

Screen Shot 2022-04-04 at 4 15 14 PM

### CODE REVIEW

**Comment by CPSTL** [comment]
Task linked: [CU-257r32t Add Coinbase wallet connection](https://app.clickup.com/t/257r32t)

**Comment by sentilesdal** [comment]
#### 📝 Changed routes:


| [**/**](https://elf-council-frontend-staging-f5pw82sdr-element-finance.vercel.app/) | [**/proposals**](https://elf-council-frontend-staging-f5pw82sdr-element-finance.vercel.app/proposals) |
|:-:|:-:|
|<a href="https://elf-council-frontend-staging-f5pw82sdr-element-finance.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/pgbqcDFnhd5jv3lHxbig91hJrgGBvRy6WY8sURqnUT46_3PQWPDurFlSJCnFrwiFV4ZaAg605IqKJOJGeAoHRg.png">(view full size)</a> |<a href="https://elf-council-frontend-staging-f5pw82sdr-element-finance.vercel.app/proposals">Screenshot of /proposals</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/CmPZb_3ilgNT0TuPp9-PWMl4m4FaqinxAMfRcJQj3gPAOUG6N8ZAPOgUaYEtYapByGucn5J4CS8Usz67Nd_QOQ.png">(view full size)</a> |







Commit 2e365f097da21f344479e279dd90b282359e6a40 (https://elf-council-frontend-staging-f5pw82sdr-element-finance.vercel.app).

File: src/elf/wallets/connectors.ts [hunk 1]
```diff
@@ -72,7 +82,7 @@ export function getConnectorName(
     return `WalletConnect (${walletBrand})`;
   }
 
-  return t`Uknown connector`;
```
Comment (cashd) [inline src/elf/wallets/connectors.ts:75]: lol

File: pages/index.tsx [hunk 2]
```diff
@@ -3,36 +3,27 @@ import React, { ReactElement } from "react";
...
-  recentDelegators: string[];
 }
-export default function Home({
-  proposalsJson,
-  recentDelegators,
-}: HomeProps): ReactElement {
+export default function Home({ proposalsJson }: HomeProps): ReactElement {
   return (
     <PageView childrenContainerClassName="flex justify-center">
-      <OverviewPage
```
Comment (cashd) [inline pages/index.tsx:18]: I actually left this file in disarray doing PRs at 3am. ty for cleaning it up!

File: src/@types/global.d.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 declare module "*.svg?url" {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const content: any;
```
Comment (cashd) [inline src/@types/global.d.ts:3]: would be nice if we could find a type for this

File: src/test/helpers/initializeGovernance.ts [hunk 4]
```diff
@@ -26,92 +26,92 @@ export async function initializeGovernance(
...
-    tokenOwner,
-  );
-  await Promise.all(
-    accounts.map((address) =>
-      // seed balances
-      tokenContract.setBalance(address, parseEther("50")),
-    ),
-  );
-}
+// async function giveAccountsVotingTokens(
```
Comment (cashd) [inline src/test/helpers/initializeGovernance.ts:29]: If we don't need, should we remove?

File: src/ui/wallet/ConnectWalletDialog.tsx [hunk 5]
```diff
@@ -45,19 +47,29 @@ export function ConnectWalletDialog({
     await activateConnector(walletConnectConnector);
   }, [activateConnector, deactivateActiveConnector]);
 
+  const handleConnectToCoinbase = useCallback(async () => {
```
Comment (cashd) [inline src/ui/wallet/ConnectWalletDialog.tsx:50]: I made this comment before, but we are duplicated logic here. We would have a generic function that takes in a connector as a parameter and handles the deactivation and activation steps.

File: src/ui/wallet/ConnectWalletDialog.tsx [hunk 6]
```diff
@@ -83,7 +106,7 @@ export function ConnectWalletDialog({
         </div>
         <Button
           variant={ButtonVariant.PALE}
-          onClick={onClose}
+          onClick={handleCloseWalletConnection}
```
Comment (cashd) [inline src/ui/wallet/ConnectWalletDialog.tsx:109]: nice nice

**Review by cashd — APPROVED** [review]
LGTM! Once we have more bandwidth (could be never) let's try to fix the lint warnings, specifically the any ones instead of ts-ignoring them :)

File: src/elf/wallets/connectors.ts (hunk 1 above)
Comment (hnguyen1179) [reply src/elf/wallets/connectors.ts:75]: lol

File: src/ui/wallet/ConnectWalletDialog.tsx (hunk 5 above)
Comment (hnguyen1179) [reply src/ui/wallet/ConnectWalletDialog.tsx:50]: makes sense, i'll make the change !

**Pull request merged**